app/src/ui/lib/conflicts/render-functions.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react'
+import { Octicon, OcticonSymbol } from '../../octicons'
+import { LinkButton } from '../link-button'
+
+export function renderUnmergedFilesSummary(conflictedFilesCount: number) {
+  // localization, it burns :vampire:
+  const message =
+    conflictedFilesCount === 1
+      ? `1 conflicted file`
+      : `${conflictedFilesCount} conflicted files`
+  return <h3 className="summary">{message}</h3>
+}
+
+export function renderAllResolved() {
+  return (
+    <div className="all-conflicts-resolved">
+      <div className="green-circle">
+        <Octicon symbol={OcticonSymbol.check} />
+      </div>
+      <div className="message">All conflicts resolved</div>
+    </div>
+  )
+}
+
+export function renderShellLink(
+  openThisRepositoryInShell: () => void
+): JSX.Element {
+  return (
+    <div>
```

Comment:
Can we return a simple fragment (`<>all the stuff</>`) here since this `div` node doesn't have an Id or class.

Reply:
I want to leave this as-is because this is how it was written for use in Merge Conflicts, and cc @outofambit for feels when she returns next week.